quickwit/quickwit-doc-mapper/src/default_doc_mapper/default_mapper.rs

```diff
@@ -156,7 +156,9 @@ fn validate_tag_fields(tag_fields: &[String], schema: &Schema) -> anyhow::Result
                     .get_indexing_options()
                     .map(|text_options| text_options.tokenizer());
 
-                if tokenizer_opt != Some(QuickwitTextTokenizer::Raw.get_name()) {
+                if tokenizer_opt.is_some()
```

Comment:
Actually, isn't that a bug? We want the raw tokenizer in the presence of a tag field, but this seems to allow `None`.